content/05.heterogeneity.md

```diff
@@ -1,22 +1,22 @@
-
-Methods like PCA, multidimensional scaling (MDS), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP) can help researchers successfully identify useful patterns in the original data, though t-SNE and UMAP may require adjusting the hyperparameters – values used to set up a model that can be specified by a user, rather than learned from the data – that may effect the output [@https://arxiv.org/abs/1802.03426; @doi:10.23915/distill.00002].
+In practice, researchers often use the first few PCs to reduce the dimensionality without removing what may be important or informative variability in the data.
+Beyond reducing the number of features in various types of data [@doi:10.1016/j.media.2020.101660; @doi:10.1038/ncomms14825], dimensionality reduction can also be used to visualize structure or artifacts in the data (e.g., [@doi:10.1038/s41467-019-13056-x]), to define sample subgroups (e.g., [@doi:10.1038/s41467-020-15351-4], or for feature selection and extraction during application of specific machine learning models [@doi:10.1007/978-3-030-03243-2_299-1] (Figure [@fig:2]c). 
+Other methods like multidimensional scaling (MDS), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP) can also help identify useful patterns in the data, though t-SNE and UMAP may require adjusting the hyperparameters that may affect the output [@https://arxiv.org/abs/1802.03426; @doi:10.23915/distill.00002].
 Testing multiple dimensionality reduction methods, rather than a single method, may be necessary to obtain a more comprehensive portrait of the data [@doi:10.1186/s13059-020-02021-3]. 
 Nguyen and Holmes discuss additional important considerations for using dimensionality reduction methods such as selection criteria and interpretation of results [@doi:10.1371/journal.pcbi.1006907].
 Beyond dimensionality reduction, other unsupervised learning approaches such as k-means clustering or hierarchical clustering have been used to characterize the structure present in genomic or imaging data [@doi:10.1186/1471-2105-9-497; @doi:10.1109/jbhi.2013.2276766].
 
-Representation learning approaches, of which dimensionality reduction methods are a subset, learns low-dimensional representations (composite features) from the raw data. 
-For example, representation learning through matrix factorization can extract features from transcriptomics datasets that are made of combinations of gene expression values found in the training data [@doi:10.1038/s41467-020-14666-6], and use them to interpret test data [@doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3].
-To ensure that the learned representations are generalizable to other data, the features learned by the model can be constrained through methods like regularization [@doi:10.1371/journal.pgen.1004754, @doi:10.1002/sim.6782]. 
```

Comment:
I think something _like this sentence_ is required _somewhere._ However, we have not yet introduced the concept of regularization by the time the reader has gotten here.

Reply:
Ah I think I included this in the model complexity section but forgot to take it away form this section.

Reply:
Line 28 of model-complexity.md refers to this .. let me know if that does not quite capture what we were trying to convey here.